R/matrixfit.R

```diff
@@ -474,7 +479,9 @@ matrixfit <- function(cf, t1, t2,
   if(is.null(cf$cf)) {
     N <- cf$N
   }
-
+  if(pcmodel) {
+    opt.tsboot[,c(1:2)] <- abs(opt.tsboot[,c(1:2)])
```

Comment:
The dimension of `opt.tsboot` is 4 × `boot.R`. Therefore this just changes the sign of all parameters (mass, mass difference, amplitude, chi²) for the first two bootstrap samples. This instead just be

```r
opt.tsboot[c(1:2), ] <- abs(opt.tsboot[c(1:2), ])
```